src/igor/power.go

```diff
@@ -139,7 +144,7 @@ func runPower(cmd *Command, args []string) {
 				log.Fatal("choked on a node named %v", n)
 			}
 
-			resID := currentSched.Nodes[index]
+			resID := currentSched.Nodes[index-1]
```

Comment:
This is the off-by-one error? `currentSched` is 0 based?

Reply:
Yep. There is likely a better way to do this, but that might take a more comprehensive approach. For now I'm just fixing the error.